server.go

```diff
@@ -174,11 +227,7 @@ func (a *App) GetContentHandler(w http.ResponseWriter, r *http.Request) {
 	rv := unpack(r)
 	meta, err := a.metaStore.Get(rv)
 	if err != nil {
-		if isAuthError(err) {
```

Comment:
auth logics are removed from meta store, and moved to app.requireAuth, see L195 above.